public/javascripts/services/surveyApi.js

```diff
@@ -0,0 +1,17 @@
+window.ChubboChat.services.surveyApi = function(userName, title, questions) {
+  var settings = {
+    url: 'https://chubbo-chat.firebaseio.com/surveys.json',
+    method: 'POST',
+    data: `{
+      "user": "`+ userName +`",
```

Comment:
this is really odd - why not use ${userName}.

also is username really 2 words, should it be camelCased ?